crates/web/Cargo.toml

```diff
@@ -9,8 +9,11 @@ crate-type = ["cdylib"]
 
 [dependencies]
 console_error_panic_hook = "0.1"
-rose = { path = "../core" }
-serde = { version = "1", features = ["derive"] }
-serde-wasm-bindgen = "0.4"
+rose = { path = "../core", features = ["serde", "serde-wasm-bindgen"] }
+serde = { version = "1", features = ["derive"], optional = true }
+serde-wasm-bindgen = { version = "0.4", optional = true }
```

Comment:
`rose-web` has `publish = false` above so, no need to make these dependencies optional